evm_arithmetization/src/cpu/kernel/tests/receipt.rs

```diff
@@ -129,7 +129,9 @@ fn test_receipt_encoding() -> Result<()> {
     // Get the expected RLP encoding.
     let expected_rlp = rlp::encode(&rlp::encode(&receipt_1));
 
-    let initial_stack: Vec<U256> = vec![retdest, 0.into(), 0.into(), 0.into()];
+    // Address at which the encoding is written.
+    let rlp_addr = U256::from(Segment::RlpRaw.unscale()) << SEGMENT_SCALING_FACTOR;
```

Comment:
nit: why unscaling to rescale after?
Could be done as 
```suggestion
    let rlp_addr = U256::from(Segment::RlpRaw. as usize);
```
I believe